src/app/components/nav/header-static/header-static.component.html

```diff
@@ -1,35 +1,46 @@
-      <li class="navlinks navlinktext fw-light" (click)="navigateTo('forum')" [class.dark-text]="headerWhite"> Forum </li>
-      <li class="navlinks navlinktext fw-light" (click)="navigateTo('join-us')" [class.dark-text]="headerWhite"> Join Us </li>
-      <!-- <li class="navlinks navlinktext">
-        <img src="assets/images/dots.png" />
-      </li> -->
-      <li class="navlinks center btn btn-nofill fw-regular" [class.btn-white-bg]="headerWhite" [class.dark-text]="headerWhite" (click)="navigateTo('/auth/signup');" *ngIf="!authState.isLoggedIn">
-        Sign Up
-      </li>
-      <li class="navlinks center btn btn-fill fw-regular" [class.btn-white-bg]="headerWhite" (click)="navigateTo('/auth/login');" *ngIf="!authState.isLoggedIn">
-        Log In
-      </li>
-      <li class="navlinks navlinktext fw-regular" (click)="navigateTo('/profile')" *ngIf="authState.isLoggedIn && authState.username" [class.dark-text]="headerWhite">
-        Hi {{authState.username}}!
-      </li>
-      <li class="navlinks center btn btn-fire fw-regular" *ngIf="authState.isLoggedIn" (click)="logOut();navigateTo('');">
-        Log Out
-      </li>
-    </ul>
-  </div>
-</header>
-<div class="overlay-container" (click)="isMenuExpanded = false;" [class.dark-overlay]="isMenuExpanded && innerWidth<=810"></div>
+<div class="main-header padding-lr-header">
+  <nav [class.grad-shadow-1]="scrolledState">
+    <div class="nav-wrapper">
+      <a (click)="menuExpander()" class="button-collapse main-header-link"><i class="fa fa-bars"></i></a>
+      <ul class="hide-on-med-and-down">
+        <li><a class="waves-effect waves-dark main-header-link evalai-logo" routerLink="/" routerLinkActive="active"><img src="assets/images/evalai-logo-single.png"></a></li>
+        <li *ngIf="!authService.isAuth"><a class="waves-effect waves-dark main-header-link" routerLink="/challenges">All Challenges</a></li>
+        <li *ngIf="!authService.isAuth"><a class="waves-effect waves-dark main-header-link" href="https://evalai-forum.cloudcv.org/" target="_blank">Forum</a></li>
+        <!-- show after authenticate user -->
+        <li *ngIf="authService.isAuth"><a class="waves-effect waves-dark main-header-link" routerLink="/dashboard" routerLinkActive="active">Dashboard</a></li>
+        <li *ngIf="authService.isAuth"><a class="waves-effect waves-dark main-header-link" href="https://evalai-forum.cloudcv.org/" target="_blank">Forum</a></li>
+      </ul>
+      <ul class="right hide-on-med-and-down">
+        <li *ngIf="!authService.isAuth"><a class="waves-effect waves-dark main-header-link" routerLink="/auth/signup" routerLinkActive="active">Sign Up</a></li>
+        <li *ngIf="!authService.isAuth"><a class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" style="padding: 0 29px; font-size: 14px; display: inline-block" routerLink="/auth/login" routerLinkActive="active">Log In</a></li>
```

Comment:
Here, as well, move `adding: 0 29px; font-size: 14px; display: inline-block` to `.scss`.

Reply:
okay